english/marshmallow.md

```diff
@@ -114,6 +114,8 @@ You also seem to be wearing an uncomfortable and restrictive jacket made of leat
 
 You don't own any uncomfortable and restrictive leather jackets... How did you end up sleeping in one?
 
+Suddenly, fuzzy memories of a tavern and an exotic woman...a house made of candy...closets full of umbrellas and leather.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.